dotnet/src/SemanticKernel.Core/Kernel.cs

```diff
@@ -113,22 +113,22 @@ public ISKFunction RegisterSemanticFunction(string pluginName, string functionNa
     }
 
     /// <inheritdoc/>
-    public IDictionary<string, ISKFunction> ImportPlugin(object pluginInstance, string? pluginName = null)
+    public IDictionary<string, ISKFunction> ImportFunctions(object functionsInstance, string? pluginName = null)
```

Comment:
I don't understand this change. Is this basically `ImportFunctionsFromPlugin` now?

Reply:
This method will import any functions defined in the class and add them to the kernel function collection as part of the specified plugin. The function instance may only be part of the plugin e.g. you could also add semnatic functions or functions from another function instance like this:

```
 IKernel target = Kernel.Builder.Build();

 target.ImportFunctions(new FooFunctions(), "MyPlugin");
 target.ImportFunctions(new BarFunctions(), "MyPlugin");

 // MyPlugin will contain the functions defined in each class
```

Reply:
Ooooooooh very cool.  How do I get `MyPlugin` though? If I take the returns, they are not equivalent, right?

```csharp
var v1 =  target.ImportFunctions(new FooFunctions(), "MyPlugin");
var v2 =  target.ImportFunctions(new BarFunctions(), "MyPlugin");

// v1 != v2, right? 
// Also concerned with how order of ImportFunctions here will change the end result of functions available. Has this been considered?
```

I guess I can do this, though:
```csharp
target.Functions.GetFunction("MyPlugin", someName);
```
But is there currently/going to be a way to browse all functions of a `Functions` instance other than `GetFunctionViews` and then need to translate back into `ISKFunction`?

Reply:
The `Kernel` instance will have the fill list of functions and these will be available at `Kernel.Functions`.

Also, you're right the order of import matters and if `FooFunctions` and `BarFunctions` both contain the same method then the last import wins.

Sergey has ideas on how to improve this and it will be covered as part of https://github.com/microsoft/semantic-kernel/issues/2828